### PR TITLE
Fix the color changing style for the contributors and hackfests layers

### PR DESCRIPTION
**Pull request by raymondnijssen**
## Description
This fixes the color changing style for the contributors and hackfests layers. Somehow I got a git conflict and now it is, again, putting my shadow change in this PR. Might have to do qgis saving xml data in different ordering every time.

I also replaced some hf locations to the right spot, as far as I could remember where they had been. And I resisted the temptation of adding extra hf information! ;-)

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and contain sufficient information in the commit message to be documented
- [ ] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [ ] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: resources/data/qgis-hackfests.json [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
+{ "type": "Feature", "properties": { "year": 2015, "month": "May", "hackfest_number": "13", "place": "Nødebo", "notes": "First QGIS User Conference. We had supper in a tipi! Forest parkour was the outside activity of choice.", "month_int": 5, "day_int": 1, "date_nice": "2015\/05\/01" }, "geometry": { "type": "Point", "coordinates": [ 12.345521450042723, 55.989587335710141 ] } },
+{ "type": "Feature", "properties": { "year": 2015, "month": "November", "hackfest_number": "14", "place": "Las Palmas", "notes": "Matthias and Marco were surfing at lunch break everyday....and not writing code during that time.", "month_int": 11, "day_int": 1, "date_nice": "2015\/11\/01" }, "geometry": { "type": "Point", "coordinates": [ -15.423848072810554, 28.110973325973482 ] } },
+{ "type": "Feature", "properties": { "year": 2016, "month": "May", "hackfest_number": "15", "place": "Girona", "notes": "Second user conference.First time we got to meet Nyall. Underwater drones!", "month_int": 5, "day_int": 1, "date_nice": "2016\/05\/01" }, "geometry": { "type": "Point", "coordinates": [ 2.833270297985213, 41.988282706951736 ] } },
```
Comment (nyalldawson) [inline resources/data/qgis-hackfests.json]: Not true - I was at nodebo a year earlier ;)

Comment (raymondnijssen) [reply resources/data/qgis-hackfests.json]: I did not write (or change) the hf notes.

(btw pls wait for another change, just found out that there is still a commented piece in one of the expressions...)

**Comment by raymondnijssen** [comment]
Done

File: resources/data/qgis-hackfests.json [hunk 2]
```diff
@@ -2,26 +2,26 @@
...
+{ "type": "Feature", "properties": { "year": 2014, "month": "October", "hackfest_number": "12", "place": "Essen", "notes": "Second Linux Hotel hackfest! We introduced plans for QGIS LTR Releases.", "month_int": 10, "day_int": 1, "date_nice": "2014\/10\/01" }, "geometry": { "type": "Point", "coordinates": [ 7.112553030272008, 51.430481213249188 ] } },
+{ "type": "Feature", "properties": { "year": 2015, "month": "May", "hackfest_number": "13", "place": "Nødebo", "notes": "First QGIS User Conference. We had supper in a tipi! Forest parkour was the outside activity of choice.", "month_int": 5, "day_int": 1, "date_nice": "2015\/05\/01" }, "geometry": { "type": "Point", "coordinates": [ 12.345521450042723, 55.989587335710141 ] } },
+{ "type": "Feature", "properties": { "year": 2015, "month": "November", "hackfest_number": "14", "place": "Las Palmas", "notes": "Matthias and Marco were surfing at lunch break everyday....and not writing code during that time.", "month_int": 11, "day_int": 1, "date_nice": "2015\/11\/01" }, "geometry": { "type": "Point", "coordinates": [ -15.423848072810554, 28.110973325973482 ] } },
```
Comment (m-kuhn) [inline resources/data/qgis-hackfests.json]: Heyyyy!! there is the information missing that we always compensated for that by staying late in the evenings!

**Pull request merged**